src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/jpa/entity/repository/UnallocatedCaseEntity.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.hmppsallocations.jpa.entity.repository
+
+import java.time.LocalDateTime
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType.IDENTITY
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "unallocated_cases")
+data class UnallocatedCaseEntity(
+  @Id
+  @Column
+  @GeneratedValue(strategy = IDENTITY)
```

Comment:
I have a vague old memory about identity not working properly on Oracle and should be base don how the column is generated.